.github/workflows/eya-def-tools-python-package.yml

```diff
@@ -17,27 +17,31 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python ${{matrix.python-version}}
       uses: actions/setup-python@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: ${{matrix.python-version}}
+    - name: Update apt-get
+      run: sudo apt-get update
+    - name: Install graphviz
+      run: sudo apt-get install graphviz graphviz-dev
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         python -m pip install --upgrade pip setuptools wheel
         pip install -r eya_def_tools/requirements.txt
         pip install -r eya_def_tools/requirements-dev.txt
+        pip install -r eya_def_tools/requirements-erd.txt
     - name: Install eya_def_tools Python package
       run: |
-        # Update this to copy JSON Schema file and not use editable install
-        pip install -e eya_def_tools[dev]
+        # Need to update this to copy JSON Schema file and not use editable install
+        pip install -e eya_def_tools[dev,erd]
     - name: Lint with flake8
       run: |
-        flake8 eya_def_tools --count --select=E9,F63,F7,F82 --show-source --statistics
-        flake8 eya_def_tools --count --exit-zero --max-complexity=14 --max-line-length=88 --extend-ignore=E203 --statistics
+        flake8 eya_def_tools --toml-config=eya_def_tools/pyproject.toml --count --show-source --statistics
     - name: Run static type checking with mypy
       run: |
         mypy --config-file eya_def_tools/pyproject.toml
     - name: Test with pytest
       run: |
-        pytest --pyargs eya_def_tools --cov=eya_def_tools --cov-report term-missing -m "not erdantic"
+        pytest --pyargs eya_def_tools --rootdir=eya_def_tools --cov-fail-under=90
```

Comment:
I included a requirement of 90% test coverage. I think it would make sense to move to 100% coverage for this package, to ensure there is no untested code. I do not think that would be too much effort to achieve.